### DISCUSSION TITLE
Sourcery 1.8 not updated in the lastest homebrew version. Please update!!!

### DISCUSSION THREAD

**Discussion by CalebSDHUB** [Homebrew/discussions#3198] [Writing Formulae/Casks]
Brew does not have the newest version of Sourcery. Please update it, thanks!

Notice that brew only provides an update for Sourcery version 1.7:

https://formulae.brew.sh/formula/sourcery

It you look at the Github webpage, the lastest Sourcery version is 1.8:

https://github.com/krzysztofzablocki/Sourcery

Thanks, for helping out!

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew
